spiffe.io/README.md

```diff
@@ -0,0 +1,13 @@
+# spiffe.io
+
+Purpose: The domain name used for the SPIFFE/SPIRE website, blog, etc.
+
+Primary Owner: @thelinuxfoundation
+
+Users/Groups with Access: Unknown
+
+Billing: Unknown
+
+URL: http://www.1api.net
```

Comment:
@drrt do you happen to know anything about our registrar? I got this link from the whois data

Reply:
This was transferred to the CNCF a while back, so whatever registrar they use

Reply:
Ok I'll chase it down with them. Thanks